core/src/main/java/hudson/AboutJenkins.java

```diff
@@ -10,7 +12,7 @@
  * 
  * @author Kohsuke Kawaguchi
  */
-@Extension
+@Extension @Symbol("about")
```

Comment:
Seems like a fairly advanced use case to me. Remember that even without `@Symbol` annotations, a DSL is able to fall back to simple or fully-qualified class names. (`SymbolLookup.find` does not currently support this, but should; `DescribableModel.resolveClass` does, though on the other hand it does not currently accept `@Symbol`!)